arlo-client/src/components/DataEntry/BoardTable.tsx

```diff
@@ -20,18 +20,21 @@ const RightWrapper = styled.div`
   }
 `
 
-const PaddedCell = styled(Cell)`
-  padding: 3px 5px;
+const Cell = styled(BPCell)`
+  padding: 7px 10px;
+  font-size: inherit;
+  > div {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 100%;
+  }
 `
 
-const GrowingTable = styled(Table)`
+const Table = styled(BPTable)`
```

Comment:
@MorganLove I was getting confused by all these custom named components that are really just styled versions of the BlueprintJS components. I would look at a component and have to track down the original BlueprintJS name in order to look up the props. So here's an experiment with a technique to let us use the original names. Let me know your thoughts.